test/loadTest/cache_refresh/reqs_x_secs.sh

```diff
@@ -56,16 +88,28 @@ while true
 do
   sleep 1s
   sec=$(($sec+1))
-  resp=`curl -s $1/receive 2>&1`
+
+  # requests received in the listener
+  resp=`curl -s $LISTENER/receive 2>&1`
   total=`echo $resp | sed 's/^{"requests": "\(.*\)","tps":\(.*\)"}/\1/'`
   tps=`echo $resp | sed 's/^{"requests": "\(.*\)","tps": "\(.*\)"}/\2/'`
+
+  # notifQueue size
+  if [ "$CB" != "None" ]
+    then
+      stat=`curl -s $CB/statistics  2>&1`
+      stat1=`echo $stat | grep -Eo 'size":[[:digit:]]+'`
```

Comment:
Not a very good idea to parse a JSON as it were a string text... However, maybe we doesn't have any option from bash.

Something like `jq` command would be greate (https://stedolan.github.io/jq). However probably it is not very much extended...